Report.md

```diff
@@ -185,42 +195,40 @@ use with the ML queues.
 
 ## Conclusion
 
-First, a reminder as to the criteria being used to compare these algorithms is:
-CPU utilization, turnaround time, waiting time, and response time. The category we care about the most is
-the maximization of CPU utilization. How the other criteria is ranked will depend on what the system is designed to do.
-Response time could be the next important if the system needs to get results back to the user rapidly even if they are
+First, a reminder as to the criteria being used to compare these algorithms are optimization of
+CPU utilization, turnaround time, waiting time, and response time. In general, the most important category is
+the maximization of CPU utilization. How the other criteria are ranked will depend on the specifics of the task at hand.
+Response time could be the next most important criterion if the system needs to get results back to the user rapidly even if they are
 only partial results. Turnaround time could be a higher priority than response time if the system depends on full
-completion of processes. The wait time, how long the process waits in the ready queue, will depend on the previous
-two design choices. The test data provided contains processes that have priorities assigned to them.
-So where appropriate that is taken into consideration.
+completion of processes. Minimizing wait time could be important if some processes are critical to the operation of the system (the CPU
+scheduler process itself, for instance). The test data provided contains processes that have priorities assigned to them, 
+and some algorithms take that into consideration.
 
 The graph [Averages by Algorithm](#average-breakdown) above details the averages of each of these criteria broken down
-by algorithm. The FCFS, SJF, and Priority were immediately disqualified due to poor CPU utilization. Which leaves us
-with MLQ, MLFQ, and Round Robin. We then contrasted Round Robin with MLFQ and decided that the lower average wait time
-and response of Round Robin were not significant enough to ignore the 3% bump on CPU utilization it achieved. So we cut
-MLFQ.
+by algorithm. The FCFS, SJF, and Priority were immediately disqualified due to poor CPU utilization, which leaves us
+with MLQ, MLFQ, and Round Robin as candidates for best choice algorithm. We then contrasted Round Robin with MLFQ and 
```

Comment:
Not your error but sense you're here. Maybe rewrite "candidates for the best choice of algorithm?" or something that makes more sense. Thanks.

Reply:
I was thinking of it like "best choice" algorithm here, but I think your suggestion works better anyway, so I used that